--- a/thread-boost-1.66.0/include/boost/thread/win32/thread_primitives.hpp
+++ b/thread-boost-1.66.0/include/boost/thread/win32/thread_primitives.hpp
@@ -49,7 +49,7 @@
             typedef ::boost::detail::winapi::HANDLE_ handle;
             typedef ::boost::detail::winapi::SYSTEM_INFO_ system_info;
             typedef unsigned __int64 ticks_type;
-            typedef ::boost::detail::winapi::FARPROC_ farproc_t;
+            //typedef ::boost::detail::winapi::FARPROC_ farproc_t;
             unsigned const infinite=::boost::detail::winapi::INFINITE_;
             unsigned const timeout=::boost::detail::winapi::WAIT_TIMEOUT_;
             handle const invalid_handle_value=::boost::detail::winapi::INVALID_HANDLE_VALUE_;
